python/lsst/ip/diffim/subtractImages.py

```diff
@@ -273,12 +274,25 @@ def run(self, template, science, sources, finalizedPsfApCorrCatalog=None):
                                             finalizedPsfApCorrCatalog=finalizedPsfApCorrCatalog)
         checkTemplateIsSufficient(template, self.log,
                                   requiredTemplateFraction=self.config.requiredTemplateFraction)
-        sciencePsfSize = getPsfFwhm(science.psf)
-        templatePsfSize = getPsfFwhm(template.psf)
+
+        fwhmExposureBuffer = self.config.makeKernel.fwhmExposureBuffer
+        fwhmExposureGrid = self.config.makeKernel.fwhmExposureGrid
+        sciencePsfSize = evaluateMeanPsfFwhm(science,
+                                             fwhmExposureBuffer=fwhmExposureBuffer,
+                                             fwhmExposureGrid=fwhmExposureGrid
+                                             )
+        templatePsfSize = evaluateMeanPsfFwhm(template,
+                                              fwhmExposureBuffer=fwhmExposureBuffer,
+                                              fwhmExposureGrid=fwhmExposureGrid
+                                              )
```

Comment:
We should perhaps have this in a try/except block as well. We try `getPsfFwhm` on the template first, and if that fails, call `evaluateMeanPsfFwhm`. For consistency, both the sizes should be computed from the same function.

Reply:
That sounds good to me. In that case, please add a log message in the except block stating that the initial PSF FWHM calculation failed.